ironfish/src/rpc/adapters/httpAdapter.ts

```diff
@@ -164,86 +167,123 @@ export class RpcHttpAdapter implements IRpcAdapter {
     response: http.ServerResponse,
     requestId: string,
   ): Promise<void> {
-    if (this.router === null || this.router.server === null) {
-      throw new RpcResponseError('Tried to connect to unmounted adapter')
-    }
-
-    const router = this.router
-
-    if (request.url === undefined) {
-      throw new RpcResponseError('No request url provided')
-    }
+    let chunkStreamed = false
 
-    this.logger.debug(
-      `Call HTTP RPC: ${request.method || 'undefined'} ${request.url || 'undefined'}`,
-    )
+    try {
```

Comment:
Wrapped this in try/catch from a .catch() on the outside so when rendering an error we can check chunkStreamed.